sites/public/src/components/listings/ListingsCombined.module.scss

```diff
@@ -30,6 +30,21 @@
     }
   }
 
+  .listings-map-skip-link {
```

Comment:
Duplicated from SiteHeader CSS but assumed this way would be preferred so we don't deepen our reliance on the local copy of UI-C